pages/common/yt-dlp.md

```diff
@@ -0,0 +1,37 @@
+# yt-dlp
+
+> A youtube-dl fork with additional features and fixes.
+> Download videos from YouTube and other websites.
+> More information: <https://github.com/yt-dlp/yt-dlp>.
+
+- Download a video or playlist (with the default options from command below):
+
+`yt-dlp "{{https://www.youtube.com/watch?v=oHg5SJYRHA0}}"`
+
+- Download best format that contains video, and if it doesn't already have an audio stream, merge it with best audio-only format (Default):
```

Comment:
Could you please reword this. I have almost no idea what it means, do you mean to say

- Download a video with the highest video and sound quality:

Reply:
@navarroaxel I understand @CleanMachine1 confusion, I was confused myself haha.
I fixed the explanation.